sympy/core/tests/test_relational.py

```diff
@@ -87,11 +88,12 @@ def test_wrappers():
 
 
 def test_Eq():
-    assert Eq(x**2) == Eq(x**2, 0)
-    assert Eq(x**2) != Eq(x**2, 1)
 
     assert Eq(x, x)  # issue 5719
 
+    with raises(SymPyDeprecationWarning):
```

Comment:
Don't use raises to test for warnings. Use the warns or (in this case) warns_deprecated_sympy context managers from sympy.utilities.pytest.

Reply:
Also add a comment that explains what is deprecated and/or links to a relevant issue.